transit_odp/fares/tests/test_querysets.py

```diff
@@ -19,31 +19,31 @@
 
 class TestFaresQuerySet:
     # Annotations
-    def test_add_published_date(self):
-        """Tests the queryset is annotated with published date"""
-        orgs = OrganisationFactory.create_batch(3)
-        DataCatalogueMetaDataFactory(
-            fares_metadata__revision__dataset__organisation=orgs[0],
-            fares_metadata__revision__is_published=False,
-        )
-        DataCatalogueMetaDataFactory(
-            fares_metadata__revision__dataset__organisation=orgs[1],
-            fares_metadata__revision__is_published=False,
-        )
-        DataCatalogueMetaDataFactory(
-            fares_metadata__revision__dataset__organisation=orgs[2],
-            fares_metadata__revision__is_published=True,
-        )
-        qs = DataCatalogueMetaData.objects.add_published_date()
+    # def test_add_published_date(self):
+    #     """Tests the queryset is annotated with published date"""
+    #     orgs = OrganisationFactory.create_batch(3)
+    #     DataCatalogueMetaDataFactory(
+    #         fares_metadata__revision__dataset__organisation=orgs[0],
+    #         fares_metadata__revision__is_published=False,
+    #     )
+    #     DataCatalogueMetaDataFactory(
+    #         fares_metadata__revision__dataset__organisation=orgs[1],
+    #         fares_metadata__revision__is_published=False,
+    #     )
+    #     DataCatalogueMetaDataFactory(
+    #         fares_metadata__revision__dataset__organisation=orgs[2],
+    #         fares_metadata__revision__is_published=True,
+    #     )
+    #     qs = DataCatalogueMetaData.objects.add_published_date()
 
-        assert len(qs) == 3
-        for org in qs:
-            if org.id == orgs[0].id:
-                assert org.last_updated_date == None
-            elif org.id == orgs[1].id:
-                assert org.last_updated_date == None
-            else:
-                assert org.last_updated_date is not None
+    #     assert len(qs) == 3
+    #     for org in qs:
+    #         if org.id == orgs[0].id:
+    #             assert org.last_updated_date == None
+    #         elif org.id == orgs[1].id:
+    #             assert org.last_updated_date == None
+    #         else:
+    #             assert org.last_updated_date is not None
```

Comment:
@pritish-ranjan01  Line 88 - Is failing for me when I run the test case (`test_add_organisation_name`):
<img width="919" alt="image" src="https://github.com/department-for-transport-BODS/bods/assets/75623814/9cb71912-9faf-4e87-b9d5-14e950481c46">